.gitmodules

```diff
@@ -10,3 +10,12 @@
 [submodule "contracts-periphery/lib/solmate"]
 	path = contracts-periphery/lib/solmate
 	url = https://github.com/Rari-capital/solmate
+[submodule "contracts-periphery/lib/v3-periphery"]
```

Comment:
Why do we have these uniswap dependencies / remappings? Those contracts are written in 0.7.6 so we can't use the contracts directly, and it seems we only use a single function's interface.

My suggestion would be remove these dependencies/remappings, and just create `src/interface/IUniswapV3SwapCallback.sol` and define the one interface we need there